core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -340,6 +340,10 @@ class EMConfig {
                     "The option has been deactivated for this run, to prevent a crash.")
             //throw IllegalArgumentException("The option to turn on Executive Summary is only meaningful when clustering is turned on (--testSuiteSplitType CLUSTERING).")
         }
+
+        if (enablePureRPCTestGeneration && outputFormat != OutputFormat.DEFAULT && !outputFormat.isJava()){
```

Comment:
what is "pure" in this contex?

Reply:
if we employ `driver`, ie. `executeRPCAction` to invoke endpoints in generated tests, or we generate it as `client.endpoint(args instance)`